ArduPlane/Plane.cpp

```diff
@@ -511,6 +511,10 @@ void Plane::update_control_mode(void)
     takeoff_state.throttle_lim_max = 100.0f;
     takeoff_state.throttle_lim_min = -100.0f;
 
+    // Preemptively set demanded pitch to NaN, so that it properly doesn't
+    // get logged. Modes ovewriting it (most modes do) will change it.
+    demanded_pitch_logged = logger.quiet_nanf();
```

Comment:
```suggestion
    demanded_pitch_logged = AP::logger().quiet_nanf();
```

Reply:
@peterbarker Why? I always assumed if we have access its better to go direct.